src/mixturemodels.jl

```diff
@@ -0,0 +1,24 @@
+function _mixlogpdf1(d::AbstractMixtureModel, x)
+    # using the formula below for numerical stability
+    #
+    # logpdf(d, x) = log(sum_i pri[i] * pdf(cs[i], x))
+    #              = log(sum_i pri[i] * exp(logpdf(cs[i], x)))
+    #              = log(sum_i exp(logpri[i] + logpdf(cs[i], x)))
+
```

Comment:
The comment seems a bit useless now that `logsumexp` is used instead of the manual implementation of the logsumexp trick.
```suggestion
```